compute/instances/preventing-accidental-vm-deletion/create_instance.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_delete_protection_create]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+// createInstance sends an instance creation request to the Compute Engine API and wait for it to complete.
+func createInstance(w io.Writer, projectID, zone, instanceName string, deleteProtection bool) error {
+	// projectID := "your_project_id"
```

Comment:
These shouldn't be commented out and include a TODO to the developer to update them.

``` golang
// TODO(developer): Replace these variables before running the sample.
projectID := "your_project_id"
```

Reply:
Same with the occurrences below.

Reply:
Oh I see. There's a mismatch with the style guide here: https://googlecloudplatform.github.io/samples-style-guide/#method-structure. We'll have to sort this out separately.